packages/create-snap/src/cli.test.ts

```diff
@@ -0,0 +1,78 @@
+import { cli } from './cli';
+import { init } from './cmds';
+
+const getMockArgv = (...args: string[]) => {
+  return ['/mock/path', '/mock/entry/path', ...args];
+};
+
+const HELP_TEXT_REGEX = /^\s*create-metamask-snap \[directory\]/u;
+
+describe('cli', () => {
+  let consoleLogSpy: jest.SpyInstance;
+  let processExitSpy: jest.SpyInstance;
+
+  beforeAll(() => {
+    global.snaps = {};
+  });
+
+  beforeEach(() => {
+    consoleLogSpy = jest.spyOn(console, 'log');
+    processExitSpy = jest.spyOn(process, 'exit');
+  });
+
+  afterAll(() => {
+    delete (global as any).snaps;
```

Comment:
Does this not work without the type assertion?

Reply:
It does not. This is the same code from snaps-cli